elrond-node.md

```diff
@@ -240,5 +240,22 @@ The `<callStack>` cell stores a list of previous contract execution states. Thes
     syntax InternalCmd ::= #exception( ExceptionCode , Bytes )
  // ---------------------------------------------------
 
+    syntax Bool ::= isBuiltinFunction(WasmStringToken)       [function, total]
+ // --------------------------------------------------------------------------
+    rule isBuiltinFunction(_) => false                          [owise]
+
+    syntax InternalCmd ::= processBuiltinFunc(String, Bytes, Bytes, VmInputCell)  [klabel(processBuiltinFunc),symbol]
+
+    syntax InternalCmd ::= checkBool(Bool, String)    [klabel(checkBool), symbol]
+ // -----------------------------------------------------------------------------------
+    rule [checkBool-t]:
+        <commands> checkBool(true, ERR) => #exception(ExecutionFailed, String2Bytes(ERR)) ... </commands>
```

Comment:
I would expect something called `checkBool` to succeed on `true` and fail on `false`. Unless I missed something, `checkFalse` or `assertFalse` would be a better name for this (making this fail on `false` would also be a good option).

Speaking of which, I checked the places where this is used. Based on the context, I would expect `checkBool(VALUE ==Int 0, ...)` to succeed when the condition is true, so I'm probably misunderstanding how this works.

Reply:
You're right. That's a terrible mistake :facepalm: While fixing this, I noticed another bug. That's why the mandos test I wrote was passing. I've fixed both issues, and added a `get/set_esdt_role` function to `testapi`.